aikau/src/main/resources/alfresco/lists/views/HtmlListView.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * 
+ * @module alfresco/lists/views/HtmlListView
+ * @extends module:alfresco/lists/views/AlfListView
+ * @author Dave Draper
+ */
+define(["dojo/_base/declare",
+        "alfresco/lists/views/AlfListView",
+        "dojo/text!./templates/HtmlListView.html",
+        "alfresco/lists/views/ListRenderer",
+        "dojo/text!./templates/HtmlListViewRenderer.html",
+        "dojo/_base/lang",
+        "dojo/_base/array",
+        "dojo/dom-construct"], 
+        function(declare, AlfListView, template, ListRenderer, rendererTemplate, lang, array, domConstruct) {
+   
+   var Renderer = declare([ListRenderer], {
```

Comment:
Note: I've created another module within the HtmlListView - it would be better for re-use if I'd abstracted this to a separate module, but for brevity I've just added it here. When we finish this off, it would be in its own module.